### ISSUE TITLE
Page titles not relevant on all pages of Docs

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#2114]
## Bug Report

**Problematic behavior**
The page title is "Docs", which does not allow identifying the current page:

```html
<title>Docs</title>
```

**Expected behavior/code**
Each page should have a relevant title in the `<title>` element, structured as follows:

```html
<title>[Nom de la page] - Docs</title>
```

**Steps to Reproduce**
1. Open any page of Docs
2. Observe the tab title in the browser
3. Open multiple Docs tabs
4. Notice that the title is not relevant or is not updated according to the context

**Critères RGAA**

**Critère 8.5 :** [Chaque page Web a-t-elle un titre de page ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#8.5)

**Critère 8.6 :** [Pour chaque page Web ayant un titre de page, ce titre est-il pertinent ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#8.6)

**Impact**
A blind user with multiple tabs open hears "Docs" for each of them without being able to distinguish the active view. They cannot quickly identify which page they are on without navigating through the content.
